students/tony7kh/hw-8/task-1.js

```diff
@@ -0,0 +1,12 @@
+function searchSubstringIndex(initialString,searchingSubstring) {
+    normalizelString = initialString.toLowerCase();
+    normalizeSubstring = searchingSubstring.toLowerCase();
```

Comment:
Please never ever ❗ ❗ ❗ declare variables without keywords. This is a bad practice and would cause an error in strict mode.
```suggestion
    const normalizelString = initialString.toLowerCase();
    const normalizeSubstring = searchingSubstring.toLowerCase();
```